manifest.json

```diff
@@ -0,0 +1,4 @@
+{
+  "name": "kuzzle-plugin-s3-upload",
```

Comment:
Very lengthy and complicated name, which makes queries a pain to write, as demonstrated in the README examples.

Opt for something simpler, such as `s3upload` (all lowercase, easier for HTTP URLs)

Also, I think there is a design problem here: I believe that this plugin should be a generic s3 plugin, with a `file` controller (or a `upload` one to separate the different kind of available actions, such as "delete"? your call)
This wouldn't change much of the current implementation, but would open the way for other capabilities at a later time.
And much more importantly, this would allow you to craft your URLs like this: `PUT https://<kuzzle>:<port>/_plugin/s3/_validate`, which seems a lot more friendly and nicely looking to me.
For other protocols, this would look like that:

```js
{
  "controller": "s3/upload",
  "action": "validate",
  ...
}
```